include/vdef.h

```diff
@@ -241,6 +241,12 @@ int __llvm_gcov_flush(void);
 #  define v_unused_
 #endif
 
+#ifdef __GNUC__
+#  define v_cold    __attribute__((cold))
+#else
+#  define v_cold
+#endif
```

Comment:
Should we use the appropriate `__GNUC_PREREQ__(x, y)` to check this one?

Reply:
Also, shouldn't it be `v_cold_` with a trailing `_` to match our other "vendor" macros?